packages/core/admin/admin/src/content-manager/components/FieldComponent.tsx

```diff
@@ -67,7 +67,7 @@ const FieldComponent = ({
 
   const allowedFields = isCreatingEntry ? createActionAllowedFields : updateActionAllowedFields;
 
-  const componentValue = modifiedData[name] ?? null;
+  const componentValue = get(name, modifiedData) ?? null;
```

Comment:
is name a dot notation in some cases then? 🤔 might be worth documenting it in that case...?

Reply:
yes it's a dot notation if the component is nested, it's why there are a few `.join('.')` and `.split('.')` in the file

I added a comment

Reply:
well that was dumb of me, great find!